purchase_requisition_order_remaining_qty/models/purchase_requisition_line.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class PurchaseRequisitionLine(models.Model):
+    _inherit = "purchase.requisition.line"
+
+    propose_qty = fields.Float(
```

Comment:
Remember to use the proper verb form:

```suggestion
    proposed_qty = fields.Float(
```